pom.xml

```diff
@@ -3,7 +3,7 @@
     <modelVersion>4.0.0</modelVersion>
```

Comment:
I can't comment on unchanged blocks so:

* The repository URL is no longer correct
* Artifact deployment configuration should be added

Reply:
You can add a comment at the end of review 😆